### PR TITLE
Documenting process

### LINKED ISSUE #235: Begin documenting design decisions
Encore-UI should have an open history of the design/architectural decisions made for the project. Some examples include things like:
- [x] Why we decided not to use Twitter Bootstrap
- [x] Why we haven't switched to Gulp
- [x] How tasks are prioritized
- [x] How it's decided to release a new version
- [x] What constitutes a 'breaking change' 

These decisions should be documented inside of Markdown files, for easy creation/consumption.

### PR DESCRIPTION
**Pull request by glynnis**
Closes #235.

Collaborated/answered many of these with @parlarjb's help. :heart:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,39 @@ If ongoing support is preferred, the EncoreUI team is available through [the Enc
 
 For access to the Slack channel, send a request through the EncoreUI mailing list.
 
+# Prioritization
+
+As a small team with limited resources, Encore relies on prioritization to drive development. However, EncoreUI is open source and new components or functionality may be submitted in a pull request. Still, there are times when EncoreUI's team must review or tweak design and functionality of features before they are ready to be brough in.
```
Comment (julianbonilla) [inline README.md]: s/brough/brought

**Comment by comeatmebro** [comment]
[Wraith UI diff](http://eb4d4842ea70055541c8-b6a3ba129b5eb76af13f4d17e4c1a53b.r51.cf5.rackcdn.com/883/gallery.html)

**Comment by comeatmebro** [comment]
[Wraith UI diff](http://eb4d4842ea70055541c8-b6a3ba129b5eb76af13f4d17e4c1a53b.r51.cf5.rackcdn.com/883/gallery.html)

File: README.md [hunk 2]
```diff
@@ -22,6 +22,39 @@ If ongoing support is preferred, the EncoreUI team is available through [the Enc
 
 For access to the Slack channel, send a request through the EncoreUI mailing list.
 
+# Prioritization
+
+As a small team with limited resources, Encore relies on prioritization to drive development. However, EncoreUI is open source and new components or functionality may be submitted in a pull request. Still, there are times when EncoreUI's team must review or tweak design and functionality of features before they are ready to be brought in.
+
+Those looking to have new features or designs prioritized by the team should attend the Encore Stakeholder Meeting, where developers, designers, contributors, and consumers of EncoreUI meet weekly to review recent releases, current work, and upcoming needs. This meeting is the best way to request support from EncoreUI for designs and features you need that haven't been developed yet.
+
+E-mail the [EncoreUI mailing list](mailto:encoreui@lists.rackspace.com) if you need an invitation to the meeting.
```
Comment (parlarjb) [inline README.md]: I'm not sure if this address is reachable from outside. If so, I'd prefer not to have it here.

File: README.md [hunk 3]
```diff
@@ -22,6 +22,39 @@ If ongoing support is preferred, the EncoreUI team is available through [the Enc
...
+
+# Common Questions about EncoreUI
+
+### Why doesn't EncoreUI utilize Twitter Bootstrap?
+
+EncoreUI is meant to serve as a framework&#8212;both architecturally and visually speaking&#8212;for all products and projects within Encore. While on occasion we have consumed a few components from Angular Bootstrap, we try to keep that to a minimum. In cases where we do utilize Angular Bootstrap, we only bring in CSS, which our designers heavily modify to ensure it has an Encore look and feel.
```
Comment (parlarjb) [inline README.md]: This isn't 100% correct. We use the javascript provided by Angular Bootstrap, but we _only bring in the CSS for the component we want to add_ (as opposed to bringing in the entirety of the CSS provided by Angular Bootstrap)

File: README.md [hunk 4]
```diff
@@ -22,6 +22,39 @@ If ongoing support is preferred, the EncoreUI team is available through [the Enc
...
+
+We have a fairly extensive build system built with grunt. Converting to gulp would require a lot of effort, and it would be difficult to validate wheter all the functionality is ported correctly, so there's no obvious benefit to making the switch.
+
+### When do releases happen? What versioning system is used?
+
+We try to release every Friday, but depending on the number of PRs ready for release (or ones that aren't quite ready), sometimes it doesn't happen and we release early the next week. If a team has an urgent need for a release, we are open to accommodating them.
+
+For versioning, we rougly follow the [Semantic Versioning system](http://semver.org/):
+ - MAJOR version when you make incompatible API changes
+ - MINOR version when you add functionality in a backwards-compatible manner
```
Comment (parlarjb) [inline README.md]: Right now we're doing incompatible API changes on `MINOR` releases. But I'm open to the release of this document being the impetus to change that. @CITguy, how do you feel about that?

File: README.md (hunk 2 above)
Comment (glynnis) [inline README.md]: FWIW, this is on the current page (i.e. before I made edits the email address was included). Should I still remove it, you think?

Comment (parlarjb) [inline README.md]: Oh, we're probably ok then. It's likely internal only.

File: README.md (hunk 3 above)
Comment (Droogans) [inline README.md]: `&#8212;` is an emdash. This is a supported character in most browsers and markdown editors. Can you just use the actual emdash character, `"—"` instead?

**Comment by Droogans** [comment]
&#8212; is an emdash. This is a supported character in most browsers and markdown editors. Can you just use the actual emdash character, "—" instead?

File: README.md (hunk 4 above)
Comment (CITguy) [inline README.md]: I'd rather follow semver: `MAJOR version when you make incompatible API changes`

File: README.md (hunk 2 above)
Comment (glynnis) [inline README.md]: Left as is.

**Comment by glynnis** [comment]
@Droogans Corrected.

Also squashed commits.

File: README.md (hunk 3 above)
Comment (glynnis) [inline README.md]: @parlarjb Updated to use your language.

**Comment by comeatmebro** [comment]
[Wraith UI diff](http://eb4d4842ea70055541c8-b6a3ba129b5eb76af13f4d17e4c1a53b.r51.cf5.rackcdn.com/883/gallery.html)

**Pull request merged**